troposphere/elasticsearch.py

```diff
@@ -55,6 +55,13 @@ class SnapshotOptions(AWSProperty):
     }
 
 
+class VPCOptions(AWSProperty):
+    props = {
+        "SecurityGroupIds": (dict, False),
```

Comment:
These should be list of strings:
    "SecurityGroupIds": ([basestring], False),
    "SubnetIds": ([basestring], False)

Reply:
You are totally right!